test/e2e_dra/upgradedowngrade_test.go

```diff
@@ -62,6 +62,8 @@ import (
 	admissionapi "k8s.io/pod-security-admission/api"
 )
 
+var ErrHTTP404 = errors.New("resource not found (404)")
```

Comment:
what's the benefit over http.StatusNotFound ?

Reply:
That's an error. If we use http status instead we'd have to artificially convert existing error returns to statuses, which would not be a good thing, e.g. what would be a status for this error: `resp, err := http.DefaultClient.Do(get)` ?

Reply:
Ack 🙃

we probably shouldn't export this though?